ct-aa/dotTheory---TypeScript
# JS-964: Next generation Javascript & Typescript

# Related Tickets
[JS-964](https://codeandtheory.atlassian.net/browse/JS-964)

# Description
- Added the arrow functions required in the tasks of this ticket.

# Validation
- Open the terminal.
- Run `npm run start`.
- Inspect the web page that will be open.
- The console log test should appear.
<img width="1774" alt="Screen Shot 2023-10-25 at 10 35 23 AM" src="https://github.com/ct-aa/dotTheory---TypeScript/assets/93116252/09f3c2ab-ed7d-41e2-800b-7c513018ae3e">

# Risks
N/A
